### PR TITLE
Updating broadcast_to function

### PR DESCRIPTION
**Pull request by uchennayah**

### CODE REVIEW

**Comment by uchennayah** [comment]
Hi @saeedashrraf, all my tests are passing, I am waiting for a review from you. Thanks

**Review by saeedashrraf — CHANGES_REQUESTED** [review]
Yes, it looks great. One last thing though, can you just remove the old occurrences broadcast_to from the project. You will most likely find it in the general.py in the folder named "old". Thank you for your efforts :)

**Review by saeedashrraf — APPROVED** [review]

**Pull request merged**
